packages/cadl-ranch-dashboard/src/components/dashboard.tsx

```diff
@@ -1,23 +1,44 @@
 import { FunctionComponent } from "react";
-import { CoverageSummary } from "../apis.js";
+import { CoverageSummary, CoverageSummaryAllTypes } from "../apis.js";
 import { DashboardTable } from "./dashboard-table.js";
+import { ResolvedCoverageReport } from "@azure-tools/cadl-ranch-coverage-sdk";
 export interface DashboardProps {
-  coverageSummary: CoverageSummary;
+  coverageSummaryAllTypes: CoverageSummaryAllTypes;
 }
 
-export const Dashboard: FunctionComponent<DashboardProps> = ({ coverageSummary }) => {
+export const Dashboard: FunctionComponent<DashboardProps> = ({ coverageSummaryAllTypes }) => {
+  const coverageSummaries: CoverageSummary[] = [];
```

Comment:
All of this should be wrapped in a `useMemo` otherwise it will do the grouping everything it rerender

Reply:
I put all of these into `useEffectAsync` of `app.tsx`.